spec/models/spree/invoice_spec.rb

```diff
@@ -0,0 +1,30 @@
+RSpec.describe Spree::Order do
+  subject { Spree::Invoice.new }
+
```

Comment:
I would also test the delegation of `invoice_number` and `invoice_date`, although you use a AR feature, it would good to now that this feature is actually used. 

Especially because the `prefix` feature is less known.